velox/core/SimpleFunctionMetadata.h

```diff
@@ -729,11 +730,24 @@ class UDFHolder {
       void,
       exec_return_type,
       const exec_arg_type<TArgs>*...>::value;
+  static constexpr bool udf_has_callNullable_return_status = util::has_method<
+      Fun,
+      callNullable_method_resolver,
+      Status,
+      exec_return_type,
+      const exec_arg_type<TArgs>*...>::value;
   static constexpr bool udf_has_callNullable =
-      udf_has_callNullable_return_bool | udf_has_callNullable_return_void;
+      udf_has_callNullable_return_bool | udf_has_callNullable_return_void |
+      udf_has_callNullable_return_status;
   static_assert(
       !(udf_has_callNullable_return_bool && udf_has_callNullable_return_void),
-      "Provided callNullable() methods need to return either void OR bool.");
+      "Provided callNullable() methods need to return either void OR bool OR Status.");
+  static_assert(
```

Comment:
I wonder if there is a way to refactor this code to replace 3 static_assert calls with just 1. Perhaps, we could define a helper function that takes 3 booleans and returns true iff only one them is true.